.clang-format

```
@@ -0,0 +1,70 @@
+# ref: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+# llvm-style: https://github.com/llvm/llvm-project/blob/main/clang/lib/Format/Format.cpp#L1175-L1365
+BasedOnStyle: LLVM
+
+ColumnLimit: 100
+IndentWidth: 2
+TabWidth: 2
+
+# Never - 탭 사용 안함
+# ForContinuationAndIndentation - 탭으로 인덴트하되 정렬을 위해 스페이스 사용
+# Always - 항상 탭 사용
+UseTab: ForContinuationAndIndentation
+
+# public, private, protected의 indent
+AccessModifierOffset: -4
```

```suggestion
AccessModifierOffset: -2
```
인덴트에 관한 값들이 탭 사이즈에 맞춰져 있어 다른 오프셋들도 변경이 필요할 것 같습니다!

---

수정했습니다! 추가로 아래 항목도 수정했습니다.
```
-ContinuationIndentWidth: 8
+ContinuationIndentWidth: 4
```